.github/workflows/publish_cpubuilder_x86_64.yml

```
@@ -4,10 +4,45 @@ on:
   push:
     branches: ['main']
     paths:
-    - dockerfiles/cpubuilder*_ghr_x86_64.Dockerfile
+    - dockerfiles/cpubuilder*_x86_64.Dockerfile
     - .github/workflows/publish_cpubuilder_x86_64.yml
 
 jobs:
+  build-cpubuilder-ubuntu-jammy:
+    runs-on: ubuntu-latest
+    env:
+      REGISTRY: ghcr.io
+      IMAGE_NAME: iree-org/cpubuilder_ubuntu_jammy_x86_64
+    # Sets the permissions granted to the `GITHUB_TOKEN` for the actions in this job.
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

We probably want to pin the action to a specific version
```suggestion
        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
```
(will also increase the score if we want to run OpenSSF scorecard, see https://github.com/ossf/scorecard/blob/main/docs/checks.md#pinned-dependencies)

---

Oh, I started switching back to using tags in https://github.com/iree-org/iree/pull/17703 

> We have been using commit hashes instead of tags, since they are more stable and that was a policy recommendation for Google-managed GitHub repos: https://opensource.google/documentation/reference/github/services#actions
> 
> > When using a third-party action (one not hosted in a Google-managed org), a fixed version of the action MUST be used by [specifying a specific commit](https://help.github.com/en/github/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepsuses), rather than a branch like "master", or a tagged release, which can be overwritten by any maintainer of the action. Docker images should always be run at a fixed version rather than "latest".
> 
> Using release tags is easier and more common, so switching back to that style.

Anyways, let's leave that for follow-up work if we want it and apply such a change to all files in the repo.